### ISSUE TITLE
Dangerous section in Terms of Use: Section 7

### ISSUE THREAD

**Issue by chlorinatedsugar** [pixelfed/pixelfed#1060]
According to section 7 of the Terms of Use, "_Pixelfed may revise these terms of service at any time without notice_". This is a dangerous term to use, since any change can occur which the user may or may not agree to. This is a violation of the user's right to choice. Since Pixelfed is supposed to be an alternative for giants that fail to respect the user's rights, this goes against the vision of many users of Pixelfed and may deter people from using the software. Even if the ones in charge are convinced to have open values, it cannot be expected from users that they may agree to any change made. Notifying the users in advance provides a safeguard against abuse.

I suggest changing it to be either of the following: either Pixelfed must inform users of _all_ changes in the Terms of Use at least _X_ days in advance, in which '_X_' is a reasonable time for users to read and acknowledge the changes; or Pixelfed must inform users of _substantial_ changes in the Terms of Use at least _X_ days in advance, which allows for more permission to change small errors in the Terms of Use without bothering users too much. I suggest using the latter option for earlier stages of development, as the Terms of Use may change quickly during this period of time. Once most of the development has finalized, and changes of the Terms of Use are rare, it should be changed to the former, so that any change is notified to the user. Of course, if the changes are small, such as grammar changes, the nature of the change can be taken into account and mentioned to the user.

**Comment by dansup** [comment]
@chlorinatedsugar As I mentioned in #1059, I used a generic terms of use generator. We will consult the community and release an improved version after we have reached a consensus.

**Comment by carbontwelve** [comment]
A pattern I have seen on a number of websites is they will first issue an email notifying users of a pending change; at which point users can pre-emptively accept or decline and delete their account. They then send a follow up email notifying the new terms now in effect and if you then log in having not pre-emptively approved the terms your login is blocked by displaying the revised terms which you must either accept or decline.

**Comment by trwnh** [comment]
Superseded by #1465

**Comment by chlorinatedsugar** [comment]
Please reopen this issue, #1465 does not supersede it. #1465 has been closed several times now, and this issue hasn't been resolved at all both times. It's currently open again, but still doesn't address this issue in any way.

**Comment by dansup** [comment]
You can add custom terms/privacy pages. Legal documents are outside the purview of this project.
